packages/monaco/src/typings/monaco/index.d.ts

```diff
@@ -1,23 +1,34 @@
+/*
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
 /// <reference types='monaco-editor-core/monaco'/>
 
 declare module monaco.instantiation {
     export interface IInstantiationService {
     }
 }
 
-declare module monaco.editor {
+declare namespace monaco.editor {
```

Comment:
@AlexTugarev Could you explain why it should be changed? (just out of curiosity)

Reply:
I think it's a left over. I just tried out something while fixing typings. The `monaco.editor` is just a global spread and it should not be necessary to declare it a module. We also never import it, right?

Reply:
@akosyakov maybe we need to change it with the transition to ESM version of monaco. I've reverted it here.

Reply:
@akosyakov  it used to be `declare module monaco` but they changed it to `declare namespace monaco`.

Reply:
it was changed sometimes between 0.11.1 and 0.11.6, I don't remember

Reply:
ok, then it makes sense :)

Reply:
I reverted it, but it won't be a problem. We'll revise it when it comes to using ESM.